scripts/copy-helpers.mjs

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env zx
```

Comment:
I've also replaced `copy-helpers.sh` with `copy-helpers.mjs` which seems easier to maintain.